src/http.c

```diff
@@ -389,18 +389,15 @@ decode_nibble(char n)
 	return n - 'A' + 10;
 }
 
-inline static char
-encode_nibble(char n)
-{
-	return n > 9 ? n + 'a' - 10 : n + '0';
-}
-
+static unsigned char
```

Comment:
How is this better?

Reply:
A 16bytes table look up operation should be much more efficient than a ternary operation, generally.

Reply:
Please see the reply in the pull request itself.